### DISCUSSION TITLE
friend's online status

### DISCUSSION THREAD

**Discussion by injox** [ppy/osu#27455] [Ideas]
In the osu!lazer friends list (as well as the website version), you can't see exactly where they are online right now. I'd like to see where they are online (stable, lazer or website) via some kind of icon

**Comment by injox** [comment]
up

**Reply by Joehuu** [reply]
Improving friend status is one of the future tasks (not high priority) of one of the core devs, see https://github.com/ppy/osu/issues/27686#issuecomment-2012410228.

**Reply by bdach** [reply]
@Joehuu just to clarify, i was giving that as an idea what could be worked on, rather than saying i will be doing so myself specifically
